Dockerfile.operator

```diff
@@ -38,6 +38,8 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 COPY --from=builder /out/${TARGETOS}/${TARGETARCH}/usr/bin/tetragon-operator /usr/bin/tetragon-operator
 WORKDIR /
 ENV GOPS_CONFIG_DIR=/
+# Avoid root privliges
+USER 65532:65532
```

Comment:
maybe instead of using a number you can create a un-privileged user by name like `operator` and use it here (making sure it has permission to access the binary)?